### PR TITLE
Update teams-recording-policy.md

### PR DESCRIPTION
**Pull request by 2kiran**

### CODE REVIEW

File: Teams/teams-recording-policy.md [hunk 1]
```diff
@@ -312,6 +312,7 @@ The following partners are in the process of certifying their solution for Micro
 
 |Partner|Solution website |
 |:--|:--|
+|Redwood Technologies |[https://www.contentguru.com/compliance-recording-for-microsoft-teams/](https://www.contentguru.com/compliance-recording-for-microsoft-teams/) |
```
Comment (JohanFreelancer9) [inline Teams/teams-recording-policy.md:315]:
```suggestion
```

File: Teams/teams-recording-policy.md [hunk 2]
```diff
@@ -312,6 +312,7 @@ The following partners are in the process of certifying their solution for Micro
...
 |GuardRec |[https://www.guardrec.com/en/teams-compliance-recording/](https://www.guardrec.com/en/teams-compliance-recording/) |
 |Landis Technologies |[https://landistechnologies.com/](https://landistechnologies.com/) |
 |Luware |[https://luware.com/en/solution/microsoft-teams-recording/](https://luware.com/en/solution/microsoft-teams-recording/) |
```
Comment (JohanFreelancer9) [inline Teams/teams-recording-policy.md:318]:
```suggestion
|Luware |[https://luware.com/en/solution/microsoft-teams-recording/](https://luware.com/en/solution/microsoft-teams-recording/) |
|Redwood Technologies |[https://www.contentguru.com/compliance-recording-for-microsoft-teams/](https://www.contentguru.com/compliance-recording-for-microsoft-teams/) |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @2kiran, kindly commit the suggested changes to keep the list in alphabetical order. Many thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@2kiran, kindly also commit the first suggested change to remove Redwood Technologies from line 315. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cabailey** [comment]
#sign-off

**Pull request merged**